js/client-battle-tooltips.js

```diff
@@ -366,13 +366,15 @@ var BattleTooltips = (function () {
 			var item = '';
 			var itemEffect = pokemon.itemEffect || '';
 			if (pokemon.prevItem) {
-				item = 'None';
 				if (itemEffect) itemEffect += '; ';
 				var prevItem = Tools.getItem(pokemon.prevItem).name;
 				itemEffect += pokemon.prevItemEffect ? prevItem + ' was ' + pokemon.prevItemEffect : 'was ' + prevItem;
 			}
+			if (itemEffect) {
+				item = '(None)';
+				itemEffect = ' (' + itemEffect + ')';
```

Comment:
Or, `(none)`.

Although in general we should probably have `(exists)` and `(none)` look more different. Suggestions?